client/src/MainLayout.tsx

```diff
@@ -1,59 +1,47 @@
 
-type Message={
-    type:string;
-    content:string;
-    sender:'user'|'ollama';
-}
-const MainLayout: React.FC = () => {
-    const [messages, setMessages] = useState<Message[]>([]);
-    const [input, setInput] = useState('');
+type MessageType = {
+  type: string;
+  content: string;
+  sender: "user" | "ollama";
+};
 
-    const handleSend = () => {
-        if (input.trim() !== '') {
-            const userMessage: Message = { type: 'text', content: input, sender: 'user' };
-            setMessages((userresponse) => [...userresponse, userMessage]);
-            setInput('');
-            const catMessage: Message = { type: 'text', content: 'I am a cat-GPT, How can I assist you?', sender: 'ollama' };
-            setMessages((userresponse) => [...userresponse, catMessage]);
-        }
-    };
+const MainLayout: React.FC = () => {
+  const [messages, setMessages] = useState<MessageType[]>([]);
+  const [input, setInput] = useState("");
 
-    return (
-        <div className="flex flex-col h-screen">
-            <header className="bg-blue-500 text-white p-5 text-center">
-                <h1 className="text-3xl font-bold">Cat-GPT</h1>
-            </header>
-            
-            <main className="flex-1 bg-blue-100 p-4 overflow-auto">
-                <div className="flex flex-col h-full space-y-4">
-                    {messages.map((msg, index) => (
-                        <Message
-                            key={index}
-                            type={msg.type}
-                            content={msg.content}
-                            sender={msg.sender}
-                        />
-                    ))}
-                </div>
-            </main>
+  const handleSend = () => {
+    if (input.trim() !== "") {
+      const userMessage: MessageType = {
+        type: "text",
+        content: input,
+        sender: "user",
+      };
+      setMessages((prevMessages) => [...prevMessages, userMessage]);
+      setInput("");
+      const catMessage: MessageType = {
+        type: "text",
+        content: "I am a cat-GPT, How can I assist you?",
+        sender: "ollama",
+      };
+      setMessages((prevMessages) => [...prevMessages, catMessage]);
+    }
+  };
 
-            <footer className="bg-white p-4 flex items-center">
-                <input
-                    type="text"
-                    className="flex-1 border border-gray-300 p-2 mr-2"
-                    placeholder="USER INPUT SECTION"
-                    value={input}
-                    onChange={(e) => setInput(e.target.value)}
-                />
-                <button 
-                   className="bg-green-500 hover:bg-green-600 text-white p-2 rounded"
-                   onClick={handleSend}>
-                    REQUEST
-                </button>
-            </footer>
-        </div>
-    );
+  return (
+    <div className="flex flex-col h-screen bg-surface">
+      <Header />
+      <main className="flex-1 p-4 overflow-auto">
```

Comment:
Shouldn't be `overflow-auto`! Can you correct this? The size of Input field should increase in height when the test surpasses line length. UPT a max of 3 lines. Than, you should apply `overflow-y` property